src/go/k8s/controllers/redpanda/cluster_controller.go

```diff
@@ -20,6 +20,7 @@ import (
 	redpandav1alpha1 "github.com/vectorizedio/redpanda/src/go/k8s/apis/redpanda/v1alpha1"
 	"github.com/vectorizedio/redpanda/src/go/k8s/pkg/labels"
 	"github.com/vectorizedio/redpanda/src/go/k8s/pkg/resources"
+	"github.com/vectorizedio/redpanda/src/go/k8s/pkg/resources/certmanager"
```

Comment:
the body of the commit doesn't tell any story. 

what does `handling` actually mean.